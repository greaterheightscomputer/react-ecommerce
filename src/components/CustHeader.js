import React from 'react';
import { Link } from 'react-router-dom';
 
const CustHeader = () => (
    <header className="admin-header">
        <div className="content-container">
            <div className="admin-header__container">                
                <Link className="admin-header__title" to="/">
                    <h1>E-Market</h1>
                </Link>
                <Link className="button button--link" to="/signin">SignIn</Link> 
            </div>
        </div>
    </header>
);

export default CustHeader;