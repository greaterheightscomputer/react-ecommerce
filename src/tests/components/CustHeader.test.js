import React from 'react';
import { shallow } from 'enzyme';
import CustHeader from '../../components/CustHeader';

test('should render CustHeader correctly', () => {
    const wrapper = shallow(<CustHeader />);
    expect(wrapper).toMatchSnapshot();
});