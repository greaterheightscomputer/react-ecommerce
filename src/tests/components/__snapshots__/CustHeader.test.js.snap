// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CustHeader correctly 1`] = `
<header
  className="admin-header"
>
  <div
    className="content-container"
  >
    <div
      className="admin-header__container"
    >
      <Link
        className="admin-header__title"
        replace={false}
        to="/"
      >
        <h1>
          E-Market
        </h1>
      </Link>
      <Link
        className="button button--link"
        replace={false}
        to="/signin"
      >
        SignIn
      </Link>
    </div>
  </div>
</header>
`;
